platform-api/src/dependencies.py

```
@@ -0,0 +1,222 @@
+    
+    # Grab the token from secrets (it might be None for public data)
+    token = settings.optional_secrets.get("sanity_api_read_token", "")
+    
+    return SanityClient(project_id=project_id, token=token)
+
+# Looks for 'X-Admin-API-Key' in the request headers
+admin_api_key_header = APIKeyHeader(name="X-Admin-API-Key", auto_error=False)
+
+async def verify_admin_api_key(
+    api_key_header: str = Security(admin_api_key_header),
+    settings: WorkerSettings = Depends(get_settings)
```

Ruff (0.15.6)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)